plugins/qlcplus_4.12.1/import.js

```diff
@@ -415,6 +432,177 @@ const qlcPlusChannelPresets = {
   })
 };
 
+const getShutterStrobeCap = (shutterEffect, speedStart, speedEnd, randomTiming) => {
+  const cap = {
+    type: `ShutterStrobe`,
+    shutterEffect
+  };
+
+  if (speedEnd) {
+    cap.speedStart = speedStart;
+    cap.speedEnd = speedEnd;
+  }
+  else if (speedStart) {
+    cap.speed = speedStart;
+  }
+
+  if (randomTiming) {
+    cap.randomTiming = true;
+  }
+
+  return cap;
+};
+
+const getRotationCapType = ({ channelName, channelType }) => {
+  if ([`Pan`, `Tilt`].includes(channelType)) {
+    return `${channelType}Continuous`;
+  }
+
+  if ([`Colour`, `Gobo`].includes(channelType) || /\bgobo\b/i.test(channelName)) {
+    return /wheel\b/i.test(channelName) ? `WheelRotation` : `WheelSlotRotation`;
```

Comment:
I won't use `\b` in both Regexes here. There will certainly be channels called `Gobowheel`, and `Wheelrotation` might also be written as one word. Anyway, which false positives to you want to sort out by using `\b`?